src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/UserTransformer.kt

```diff
@@ -102,4 +105,19 @@ class UserTransformer(
     UserQualification.MENTAL_HEALTH_SPECIALIST -> ApiUserQualification.mentalHealthSpecialist
     UserQualification.RECOVERY_FOCUSED -> ApiUserQualification.recoveryFocused
   }
+
+  private fun transformApprovedPremisesRoleToPermissionApi(userRole: UserRoleAssignmentEntity): List<ApiUserPermission> {
+    var listOfPermissions = mutableListOf<ApiUserPermission>()
+    userRole.role.permissions.forEach {
+      listOfPermissions += when (it) {
+        UserPermission.CAS1_VIEW_ASSESSMENTS -> ApiUserPermission.viewAssessments
+        UserPermission.CAS1_PROCESS_AN_APPEAL -> ApiUserPermission.processAnAppeal
+        UserPermission.CAS1_ASSESS_PLACEMENT_APPLICATION -> ApiUserPermission.assessPlacementApplication
+        UserPermission.CAS1_ASSESS_PLACEMENT_REQUEST -> ApiUserPermission.assessPlacementRequest
+        UserPermission.CAS1_ASSESS_APPLICATION -> ApiUserPermission.assessApplication
+        UserPermission.CAS1_ASSESS_APPEALED_APPLICATION -> ApiUserPermission.assessAppealedApplication
+      }
+    }
```

Comment:
instead of using a mutable map you could maybe do something like the following:

```
return userRole.role.permissions.map {
   when(it) {
     ... existing code here ...
   }
}
```